Formula/vivid.rb

```diff
@@ -0,0 +1,17 @@
+class Vivid < Formula
+  desc "Generator for LS_COLORS with support for multiple color themes"
+  homepage "https://github.com/sharkdp/vivid"
+  url "https://github.com/sharkdp/vivid/archive/v0.6.0.tar.gz"
+  sha256 "c8640f524aef1cd4dc15286bdc6189894ad067ea79bf8c40b9ca8d9d752d161f"
+  license any_of: ["MIT", "Apache-2.0"]
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert !shell_output("#{bin}/vivid generate molokai").empty?
```

Comment:
Please check for some actual output here instead of just non-empty.

Reply:
Is it acceptable to give the program an invalid input and expect an error?

Reply:
Yeah, that's fine. Here's my default explanation for tests.

---

We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
@SimplyDanny try this instead:
`assert_includes shell_output("#{bin}/vivid preview molokai"), "archives.images: \e[4;38;2;249;38;114m*.bin\e[0m\n"`

Reply:
Changed. I actually had this before the simpler `!.empty?`. 😊